cmd/podman/main.go

```diff
@@ -169,10 +169,6 @@ func main() {
 			Name:  "cni-config-dir",
 			Usage: "path of the configuration directory for CNI networks",
 		},
-		cli.StringFlag{
```

Comment:
Why are we dropping this?

Reply:
It is not used by anything that I could see.